browser/ui/BUILD.gn

```diff
@@ -176,10 +177,6 @@ source_set("ui") {
     sources += [
       "webui/settings/brave_settings_import_data_handler_mac.mm",
     ]
-  } else {
```

Comment:
I think this entire block needs to move up  to https://github.com/brave/brave-core/pull/4711/files#diff-4298e412eb456ad3ceed2346abb95f6aR99 to avoid building brave_settings_import_data_handler.cc in macos